src/models/SavedItem.ts

```diff
@@ -60,6 +60,54 @@ export class SavedItemModel {
     return savedItem;
   }
 
+  /**
+   * 'Favorite' a Save in a Pocket User's list
+   * @param id the ID of the SavedItem to favorite
+   * @param timestamp timestamp for when the mutation occurred. Optional
+   * to support old id-keyed mutations that didn't require timetsamp.
+   * If not provided, defaults to current server time.
+   * @returns The updated SavedItem if it exists, or null if it doesn't
+   * @throws NotFound if the SavedItem doesn't exist
+   */
+  public async favoriteById(
+    id: string,
+    timestamp?: Date
+  ): Promise<SavedItem | null> {
+    const savedItem = await this.saveService.updateSavedItemFavoriteProperty(
+      id,
+      true,
+      timestamp
+    );
+    if (savedItem == null) {
```

Comment:
should we use `===` if we are strictly checking if it's `null` 🤔

Reply:
thanks for the comment, typically when there isn't a behavior difference between null and undefined I use `==` because it's less prone to developer error bugs